### ISSUE TITLE
False positive on “Internet Explorer”

### ISSUE THREAD

**Issue by sapegin** [sapegin/textlint-rule-terminology#14]
> …but there are no reasons to use this anymore: Internet Explorer 8 was the last browser that didn’t support trailing commas.

```
2295:178  ✓ error  Incorrect usage of the term: “anymore: Internet”, use “anymore: internet” instead  terminology
```

**Comment by sapegin** [comment]
Should be fixed by #20.
